docs/concepts/what-is-asynchronous-activity-completion.md

```diff
@@ -0,0 +1,14 @@
+---
+id: what-is-asynchronous-activity-completion
+title: What is Asynchronous Activity Completion?
+sidebar_label: Asynchronous Activity Completion
+description: Asynchronous Activity Completion is the scenario where the final result of a computation, started by an Activity, is provided to the Temporal Platform through the Temporal Client.
```

Comment:
```suggestion
description: Asynchronous Activity Completion occurs when an external system provides the final result of a computation, started by an Activity, to the Temporal System.
```
Un-passive-fying* the sentence. (*Of _course_ that's a word. Why wouldn't it be a word?)